src/screens/Notifications/ConfirmEmailScreen.tsx

```diff
@@ -0,0 +1,89 @@
+import { CompositeNavigationProp, RouteProp } from '@react-navigation/native';
+import { StackNavigationProp } from '@react-navigation/stack';
+import React, { Component } from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+
+import { Header, ScreenTemplate, Button, FlatButton, CodeInput } from 'app/components';
+import { Route, MainCardStackNavigatorParams, RootStackParams } from 'app/consts';
+import { typography, palette } from 'app/styles';
+
+const i18n = require('../../../loc');
+
+interface Props {
+  navigation: CompositeNavigationProp<
+    StackNavigationProp<RootStackParams, Route.MainCardStackNavigator>,
+    StackNavigationProp<MainCardStackNavigatorParams, Route.ConfirmEmail>
+  >;
+  route: RouteProp<MainCardStackNavigatorParams, Route.ConfirmEmail>;
+}
+
+interface State {
+  code: string;
+}
+
+export class ConfirmEmailScreen extends Component<Props, State> {
+  state = {
+    code: '',
+  };
+
+  onConfirm = () => {};
+
+  onChange = (code: string) => this.setState({ code });
+
+  onResend = () => {};
```

Comment:
I assume it's empty only because the back-end isn't ready yet, right?

Reply:
But actually, onResend there should be a logic of disabling the button and displaying countdown timer as far as I know. Will it be done in another PR?

Reply:
There will be an error message instead of countdown if get it correctly. So yes it will be filled when adding logic